biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -698,7 +698,7 @@ public void close() throws IOException {
 		toBeClosed.clear();
 	}
 
-	private void clearPlugins() {
+	public void clearPlugins() {
```

Comment:
I do not think this is a good idea to make this public. Please keep private for internal use by Processor.